pkg/ebpf/netlink/conntracker.go

```diff
@@ -13,16 +13,20 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/process/util"
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 	ct "github.com/florianl/go-conntrack"
+	"github.com/mdlayher/netlink"
 	"github.com/pkg/errors"
 )
 
 const (
 	initializationTimeout = time.Second * 10
 
-	compactInterval = time.Minute * 3
+	compactInterval = time.Minute
 
-	// generationLength must be greater than compactInterval to ensure we have  multiple compactions per generation
-	generationLength = compactInterval + time.Minute
+	// generationLength must be greater than compactInterval to ensure we have multiple compactions per generation
+	generationLength = compactInterval + 30*time.Second
```

Comment:
I know that this isn't new but would you add some documentation around what this field does ?

Reply:
happy to do it in a future PR as I'm trying to get this fix shipped ASAP and the pipeline takes way too long to complete.